webpack.config.js

```diff
@@ -0,0 +1,21 @@
+/* global require, module, __dirname */
+
+const path = require("path");
+
+module.exports = {
+  entry: {
+    feature: "./feature.js/index.js",
```

Comment:
It seems strange to me to have folders with file extensions in their name. Is this a webpack design pattern?

Reply:
Yeah, I believe so. All the files inside the folder are "packed" into feature.js. I got the pattern from @motin's repos :)